typescript/src/schema/proto.ts

```diff
@@ -288,6 +289,23 @@ function toLRInterface(type: string, inputType: string) {
   return inputType.replace(/\.([^.]+)$/, '.I' + type);
 }
 
+function getHeaderParms(rule: plugin.google.api.IHttpRule): string {
+  const message = rule.post
```

Comment:
Is this the same as `rule.post || rule.delete || rule.get || rule.put || rule.patch`?

Reply:
it works, thanks! was making it too complicated